sdk/cosmos/azure-spring-data-2-3-cosmos/README.md

```diff
@@ -1,9 +1,73 @@
 # Azure Spring Data 2.3 Cosmos client library for Java
 
-## Key concepts
+**Azure Spring Data Cosmos** provides Spring Data 2.3.x support for Azure Cosmos DB using the [SQL API][sql_api_query], based on Spring Data framework. 
+[Azure Cosmos DB][cosmos_introduction] is a globally-distributed database service which allows developers to work with data using a variety of standard APIs, such as SQL, MongoDB, Cassandra, Graph, and Table.
+
+[Source code][source_code] | [Package (Maven)][azure_spring_data_2_3_cosmos_maven] | [API reference documentation][api_documentation] | [Product documentation][azure_spring_data_cosmos_docs] |
+[Samples][samples]
+
+## Spring data version support
+This project supports [spring-data-commons 2.3.x][azure_spring_data_2_3_cosmos] version.
```

Comment:
This should link to **spring-data-commons** instead of **azure_spring_data_2_2_cosmos**?

Reply:
Yes, fixed it.